forms/FormField.php

```diff
@@ -138,6 +138,8 @@ public static function name_to_label($fieldName) {
 	 * @return string
 	 */
 	public static function create_tag($tag, $attributes, $content = null) {
+		Requirements::javascript(FRAMEWORK_DIR . '/javascript/FieldToggleDescription.js');
```

Comment:
Also make sure that's included in the main `Requirements::combine_files()` call in `CMSMain`, we don't want to cause more asset loads to slow down the CMS